index.html

```diff
@@ -0,0 +1,795 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Raleway:wght@200;300;400;700;800;900&display=swap">
+  <link rel="stylesheet" href="assets/css/style.css">
+  <link rel="stylesheet" href="assets/css/style768.css" media="screen and (min-width: 768px)">
+  <link rel="stylesheet" href="assets/css/style1024.css" media="screen and (min-width: 1024px)">
+
+  <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+
+  <title>Responsive design clone of the next web homepage.</title>
+</head>
+<body>
+  <header>
+    <figure class="logo"><img src="assets/img/tnw.svg" alt=""></figure>
```

Comment:
This is just an example but there are other images that are missing a description on the alternative text attribute.
Actual
`<figure class="logo"><img src="assets/img/tnw.svg" alt=""></figure>`
** Improved** +1 :
`<figure class="logo"><img src="assets/img/tnw.svg" alt="TNW Logo"></figure>`
Having an alternate text help SEO and Screen-readers